robottelo/cli/factory.py

```diff
@@ -2679,14 +2681,30 @@ def setup_capsule_virtual_machine(capsule_vm, org_id=None, lce_id=None,
         except CLIReturnCodeError as err:
             raise CLIFactoryError(
                 u'Failed to fetch repository info\n{0}'.format(err.msg))
-    # Synchronize the RH repositories
-    for rh_repo in rh_repos:
+    # If we aren't working with CDN, create custom repo with latest capsule
+    # repo available
+    if not settings.cdn and settings.capsule_repo:
```

Comment:
i thought about this, it won't be user-friendly - i suspect some of our folks don't have `capsule_repo` and `sattools_repo` settings set, by introducing exception we'll block them till they update properties file. And also for robottelo users outside our team this change will basically mean 'if you don't want to run the tests against cdn, you have to have downstream repos for satellite tools and satellite capsule' which is probably not the best approach.
Basically, these lines mean 'if we're in downstream then try to use internal downstream build if available, if not - at least use latest released one' (which will work in most cases btw).